QueryEngine/RelAlgTranslator.cpp

```diff
@@ -178,14 +178,22 @@ std::shared_ptr<Analyzer::Expr> RelAlgTranslator::translateAggregateRex(
     const std::vector<std::shared_ptr<Analyzer::Expr>>& scalar_sources) {
   const auto agg_kind = rex->getKind();
   const bool is_distinct = rex->isDistinct();
-  const auto operand = rex->getOperand();
-  const bool takes_arg{operand >= 0};
+  const bool takes_arg{rex->size() > 0};
+  std::shared_ptr<Analyzer::Expr> arg_expr = nullptr;
```

Comment:
You don't need the initialization to `nullptr`, the default constructor of `std::shared_ptr` does the right thing.